share-seq.wdl

```diff
@@ -130,6 +131,72 @@ workflow ShareSeq {
             windowPadSize = windowPadSize,
             mem_gb = mem_gb_dorcs
     }
+    call html_report.html_report as html_report {
+        input:
+            atac_total_reads = atac.atac_total_reads,
+            atac_aligned_uniquely = atac.atac_aligned_uniquely,
+            atac_unaligned = atac.atac_unaligned,
+            atac_feature_reads = atac.atac_feature_reads,
+            atac_duplicate_reads = atac.atac_duplicate_reads,
+            rna_total_reads = rna.rna_total_reads,
+            rna_aligned_uniquely = rna.rna_aligned_uniquely,
+	    rna_aligned_multimap = rna.rna_aligned_multimap,
+            rna_unaligned = rna.rna_unaligned,
+            rna_feature_reads = rna.rna_feature_reads,
+            rna_duplicate_reads = rna.rna_duplicate_reads,   
+
+            ## JPEG files to be encoded and appended to html
+            # RNA plots
+            share_rna_qc_library_plot = rna.share_rna_qc_library_plot,
+            share_rna_seurat_raw_violin_plot = rna.share_rna_seurat_raw_violin_plot,
```

Comment:
example for line 145 if you decide to change the name in the task `rna_feature_reads = rna.share_rna_feature_reads`

suggestion for line 150, you could reduce that to  `rna_qc_library_plot = rna.share_qc_library_plot`